gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Tue Jan 30 21:06:06 MST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```

Comment:
Did you intend to commit this file too? If not, would you mind reverting just this file? This can be the topic of another commit if you feel the change is needed. Thanks! :-)

Reply:
No. Android Studio updated this one itself. I will fix this. But Let's close this pull request and I will make a new one with all the string files update. deal?